cmake/Findhiredis.cmake

```diff
@@ -60,6 +60,7 @@ if(do_download)
     URL https://github.com/redis/hiredis/archive/refs/tags/v${hiredis_version}.tar.gz
     SOURCE_DIR ${hiredis_dir}
     BINARY_DIR ${hiredis_build}
+    DOWNLOAD_EXTRACT_TIMESTAMP TRUE
```

Comment:
Looks like this was introduced in CMake 3.24? What happens if the user has an older CMake?

Reply:
It does in fact break cmake 3.18 on Debian 11, good catch there.

Reply:
Fixed.